README.md

```diff
@@ -1,34 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Meals Recipes - API
 
-> Description the project.
+> We collaborated @adel @svitalis123 to build a Meals Recipe API to help the client access delicious foods right at their technologies.
 
+## Desktop preview 
+![desktop-version](https://user-images.githubusercontent.com/68030297/197234750-cb72dd94-d542-416c-89c2-4e5ccda8a36e.GIF)
+
+## Mobile preview 
+![mobile-version](https://user-images.githubusercontent.com/68030297/197234829-646f2011-fddc-4016-9849-f2a94a48c84d.GIF)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML5
+- CSS3
+- Bootstrap
+- JavaScript
+- Webpack
 
-## Live Demo (if available)
-[Experience a live Demo 🚀](https://livedemo.com)
+## Live Demo 
+[Experience a live Demo 🚀](https://adel-gu.github.io/Meals-API/)
 
+## A brief description on the project
+[Video description](https://drive.google.com/file/d/1cW2jRlLERSZEJq7JpNkH4vNYuovwiHtB/view?usp=sharing)
 
 ## Getting Started
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
+- Clone this repo using this URL (link) or download it as a ZIP format from the green button above under the text Code.
 
 ### Install
-
-### Usage
+```shell
+npm install
+```
 
 ### Run tests
+```shell
+npm run test
+```
 
 ### Deployment
+```shell
+npm run deploy
+```
 
 ## Author
```

Comment:
- [ ] OPTIONAL
I think you should change the header title _Author_ to _Authors_, as this project is worked on by two contributors.

Reply:
Definitely we will fix that, thank you  👏